ElectronClient/app/gui/SideBar.jsx

```diff
@@ -336,11 +336,11 @@ class SideBarComponent extends React.Component {
 		if (itemType === BaseModel.TYPE_FOLDER && !item.encryption_applied) {
 			menu.append(
 				new MenuItem({
-					label: _('Rename'),
+					label: _('Properties'),
 					click: async () => {
 						this.props.dispatch({
 							type: 'WINDOW_COMMAND',
-							name: 'renameFolder',
+							name: 'commandNotebookProperties',
```

Comment:
Maybe just `openNotebookProperties` as the "command" part is implied.

Reply:
Thanks. That might be the issue. Currently, I'm removing the serialization code because it's throwing Travis build errors

Reply:
So, I just checked into this. If you look at ElectronClient/app/gui/MainScreen.jsx, where I placed this particular command, there are multiple command named WINDOW_COMMAND's. I assumed this was a deliberate naming convention as part of the design, so I stuck with it. I can definitely change if you need me to.

Reply:
Sent a commit for this